package.json

```diff
@@ -85,7 +87,8 @@
     "lint": "eslint 'src/**/*.{js,ts,tsx}' 'stories/**/*.{js,ts,tsx}' --fix",
     "lint:ci": "eslint 'src/**/*.{js,ts,tsx}' 'stories/**/*.{js,ts,tsx}'",
     "build-storybook": "storybook build",
-    "prepublishOnly": "yarn lint:ci && yarn test:ci && yarn storybook --smoke-test"
+    "prepublishOnly": "yarn lint:ci && yarn test:ci && yarn storybook --smoke-test",
+    "chromatic": "npx chromatic --exit-zero-on-changes"
```

Comment:
If we use their action we can remove this. 

Or if we are keeping it make sure to add @latest 

CI pulled in v6 and v10.xx is latest 

Or even better add it as a devDep